specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/examples/AutoDismissRules/GetAutoDismissRule_example.json

```diff
@@ -0,0 +1,40 @@
+{
+  "parameters": {
+    "api-version": "2019-01-01-preview",
+    "subscriptionId": "20ff7fc3-e762-44dd-bd96-b71116dcdc23",
+    "autoDismissRuleName": "dismissIpAnomalyAlerts"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/providers/Microsoft.Security/autoDismissRules/dismissIpAnomalyAlerts",
+        "name": "dismissIpAnomalyAlerts",
+        "type": "Microsoft.Security/autoDismissRules",
+        "properties": {
+          "alertType": "IpAnomaly",
+          "lastModifiedUTC": "2019-07-31T19:50:47.083633Z",
+          "expirationDateUTC": "2019-12-01T19:50:47.083633Z",
+          "state": "Enabled",
+          "reason": "Detects a normal activity (False Positive)",
```

Comment:
Do we want to convert Reason into some kind of Enum? so the user will have easier time to send it?

Reply:
The reasons probably will changed from time to time so we validate this in RP